docs/docs-content/automation/palette-cli/commands/validate-auth.md

```diff
@@ -0,0 +1,351 @@
+| -------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+|                | `--cloud-env` | The target cloud environment to validate permissions on. This can also be selected in the interactive wizard. The supported cloud environments are `aws` and `azure`. | string   |
+|                | `--outdir`    | Output directory for generated resources. Defaults to `$(pwd)/out`.                                                                                                   | string   |
+| `-h`           | `--help`      | Help with any command.                                                                                                                                                | -        |
+
+### Examples
+
+Here are some examples of using the `validate-auth` command and its supported flags.
+
+Start the interactive steps.
+
+```shell
+palette validate-auth
+```
+
+Start the interactive steps and choose the cloud environment in advance.
+
+```shell
+palette validate-auth --cloud-env <aws|azure>
+```
+
+Start the interactive steps and specify the output directory for any generated resources.
+
+```shell
+palette validate-auth --outdir $(pwd)/resources
+```
+
+### Interactive Steps
+
+The interactive steps change depending on the cloud environment chosen.
+
+#### AWS
+
+1. Issue the `validate-auth` command using the Palette CLI.
+
+   ```shell
+   palette validate-auth
+   ```
+
+2. When prompted, select `AWS` for the cloud environment, and press Enter.
+
+3. Provide the AWS Access Key ID for the IAM user that will perform the validation, and press Enter.
+
+4. Provide the AWS Secret Access Key for the IAM user that will perform the validation, and press Enter.
+
+5. Select either `IAM User` or `IAM Role` depending on what you want to check, and press Enter.
+
+6. Provide the `IAM User name` or `IAM Role name` depending on what was chosen in step 5, and press Enter. This is the
+   user or role that will be deploying clusters in your environment.
+
+7. Choose the permission model to validate against. The `Comprehensive` model will check that whether you have
+   sufficient permissions for all Palette features. The `Minimal` model will only check for the privileges needed to
+   create and delete clusters.
+
+   Once you have selected the permission model, press Enter.
+
+8. You are prompted to answer `Will the cloud account be deploying EKS host clusters?`. Enter `y` if you want to check
+   for sufficient permissions to deploy Amazon EKS clusters, or `n` if you do not.
+
+9. Provide the default AWS region that the validation will be performed on. This does not have to match the region where
+   you intend to deploy clusters. Press Enter after providing the region.
```

Comment:
We should tell users to aim at the region they want to deploy clusters to. Service quotas vary by AWS region so by checking the target region, they can catch issues sooner.